test/testdata/fixtures/multi-dc-auth/k8ssandra.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   auth: false
   cassandra:
-    serverVersion: "4.0.1"
+    serverVersion: "3.11.11"
```

Comment:
same as before, it seems like 4.0 shows a heavier footprint in our tests and switching some flaky tests to 3.11 helped stabilize the executions.
I'm not satisfied with it as we'll quickly need to run both versions at the same time using a matrix, but it can be considered outside of the scope of this work (single cluster e2e tests are still running on 4.0 FTR).